### PR TITLE
Add Transit CMAC stubs in CE

### PR DESCRIPTION
**Pull request by stevendpclark**
Add stubs within CE to support the Transit CMAC enterprise functionality.

### CODE REVIEW

File: builtin/logical/transit/path_sign_verify.go [hunk 1]
```diff
@@ -226,6 +232,11 @@ derivation is enabled; currently only available with ed25519 keys.`,
 				Description: `Hash algorithm to use (POST URL parameter)`,
 			},
 
+			"mac_length": {
+				Type:        framework.TypeInt,
+				Description: `MAC length to use (POST body parameter). Valid values are:`,
```
Comment (sgmiller) [inline builtin/logical/transit/path_sign_verify.go:237]: Where do these values get specified?

File: builtin/logical/transit/path_sign_verify.go [hunk 2]
```diff
@@ -542,38 +556,48 @@ func (b *backend) pathVerifyWrite(ctx context.Context, req *logical.Request, d *
...
 			sigFound = true
 		} else if _, ok := v["hmac"]; ok {
 			hmacFound = true
+		} else if _, ok := v["cmac"]; ok {
+			cmacFound = true
 		} else {
 			missing = true
 		}
 	}
+	optionsSet := numBooleansTrue(sigFound, hmacFound, cmacFound)
```
Comment (sgmiller) [inline builtin/logical/transit/path_sign_verify.go:572]: Cute

**Review by sgmiller — COMMENTED** [review]
Just the one Q

File: builtin/logical/transit/path_sign_verify.go (hunk 1 above)
Comment (stevendpclark) [reply builtin/logical/transit/path_sign_verify.go:237]: They are processed solely on the enterprise side within `parseCmacVerifyInput` as an individual POST parameter or as a parameter within the `batch_input`. They just need to be defined here as we piggy back on the CE /verify api call.

Comment (sgmiller) [reply builtin/logical/transit/path_sign_verify.go:237]: Got it.

**Review by sgmiller — APPROVED** [review]

**Pull request merged**